apps/api/src/modules/auth/dto/login-response.ts

```diff
@@ -7,4 +7,7 @@ export class LoginResponse {
 
   @Field()
   user: string;
+
+  @Field(() => [String], { nullable: true })
+  allKeys?: string[];
```

Comment:
Should we instead have a `ServerApiKey` entity object here instead?

Reply:
Changed it to list of ServerApiKey